introduction.md

```diff
@@ -3,3 +3,8 @@ There's a saying: When the student is ready the master will come...
 I have many names, but in the www universe they call me gnzandrs!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?